services/blockchain-coordinator/jobs/missingBlocks.js

```diff
@@ -27,10 +27,12 @@ module.exports = [
 		controller: async () => {
 			logger.debug('Schedule missing blocks indexing...');
 			try {
+				// TODO: Do not schedule if the queue has jobs more than a specified threshold
 				await scheduleMissingBlocksIndexing();
-				logger.info('Successfully scheduled missing blocks indexing');
+				logger.info('Successfully scheduled missing blocks indexing.');
 			} catch (err) {
-				logger.warn(`Schedule missing blocks indexing failed due to: ${err.message}`);
+				logger.warn(`Schedule missing blocks indexing failed due to: ${err.message}.`);
```

Comment:
Should we update this comment?

```suggestion
				logger.warn(`Failed to schedule missing blocks indexing due to: ${err.message}.`);
```